test/unit/rules/require-input-label-test.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const { ERROR_MESSAGE } = require('../../../lib/rules/require-input-label');
+const { ERROR_MESSAGE, ERROR_MESSAGE_MULTIPLE_LABEL } = require('../../../lib/rules/require-input-label');
```

Comment:
```suggestion
const { 
  ERROR_MESSAGE,
  ERROR_MESSAGE_MULTIPLE_LABEL
} = require('../../../lib/rules/require-input-label');
```

I think this is what prettier is complaining about @rwjblue